src/stores/Annotation/Annotation.js

```diff
@@ -978,6 +977,23 @@ export const Annotation = types
         if (tagNames.has(obj.from_name) && tagNames.has(obj.to_name)) {
           res.push(obj);
         }
+        
+        // Insert image dimensions from result 
+        (() => {
+          if (!obj.type.endsWith('labels')) return;
```

Comment:
They can be just a usual tool result, like 'rectangle'. So I'd look specifically for `original_width`, because if they are not here, which is possible it would also be bad.